modules/setting/setting.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/gogits/gogs/modules/bindata"
 	"github.com/gogits/gogs/modules/log"
 	"github.com/gogits/gogs/modules/user"
+
+	"github.com/strk/go-libravatar"
```

Comment:
3rd-party imports in the wrong place, ref: https://github.com/Unknwon/go-code-convention/blob/master/en-US/import_packages.md